### PR TITLE
Di 23000 account employees should have field from erp integration

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: server-side/installation.ts [hunk 1]
```diff
@@ -150,6 +150,24 @@ export async function upgrade(client: Client, request: Request): Promise<any>
 			return res;
 		}
 	}
+	// Add FromERPIntegration field to account_employees
+	else if(request.body.FromVersion && semverLessThanComparator(request.body.FromVersion, '0.7.5'))
```
Comment (ShirMarciano) [inline server-side/installation.ts:154]: why this is not in the above semver check on 0.7.2?

File: server-side/api.ts [hunk 2]
```diff
@@ -65,6 +70,11 @@ export async function account_users(client: Client, request: Request)
 	return await resourcesFunctionAdapter(client, request, "account_users");
 }
 
+export async function account_employees(client: Client, request: Request)
```
Comment (ShirMarciano) [inline server-side/api.ts:73]: I was sure this was already done, wierd

File: server-side/installation.ts (hunk 1 above)
Comment (saargt) [reply server-side/installation.ts:154]: @ShirMarciano  why would it be? The functionality is presented in 0.7.5 - what would happen if you had 0.7.4 an upgraded to 0.7.5?

Comment (ShirMarciano) [reply server-side/installation.ts:154]: Didn't thought about the QA - this version is not phased and no customer have it that why I asked

**Pull request merged**